root/static/scripts/guess-case/MB/GuessCase/Main.js

```diff
@@ -1,7 +1,6 @@
 /*
    This file is part of MusicBrainz, the open internet music database.
-   Copyright (c) 2005 Stefan Kestenholz (keschte)
```

Comment:
You shouldn't remove keschte's copyright line, unless you've removed all the code he's written in the file.